toolchain/check/eval.cpp

```diff
@@ -1806,6 +1813,33 @@ auto TryEvalTypedInst<SemIR::BindSymbolicName>(EvalContext& eval_context,
   return MakeConstantResult(eval_context.context(), bind, phase);
 }
 
+static auto IsPeriodSelf(EvalContext& eval_context, SemIR::ConstantId const_id)
+    -> bool {
+  if (!const_id.is_symbolic()) {
+    return false;
+  }
+  const auto& symbolic_info =
```

Comment:
```suggestion
  const auto& symbolic =
```

(info is not part of the type, and doesn't add anything extra to the variable name)

Reply:
Okay, but I had just copied the other call to GetSymbolicConstant from this file.